sdk/src/main/java/io/verloop/sdk/VerloopNotification.kt

```diff
@@ -5,19 +5,35 @@ import android.app.NotificationManager
 import android.app.PendingIntent
 import android.content.Context
 import android.content.Intent
+import android.content.pm.PackageManager
 import android.os.Build
 import android.util.Log
 import androidx.annotation.DrawableRes
 import androidx.core.app.NotificationCompat
 import io.verloop.sdk.Verloop.Companion.isActivityVisible
 import org.json.JSONException
 import org.json.JSONObject
-import android.content.pm.PackageManager
-
 
 object VerloopNotification {
+
     private const val TAG = "VerloopNotification"
 
+    @JvmStatic
+    @Deprecated(
+        "Use showNotification with channelName parameter to overwrite default channelName",
+        ReplaceWith(
+            "showNotification(context, smallIcon, data, channelName)",
+            "io.verloop.sdk.VerloopNotification.showNotification"
+        )
+    )
+    fun showNotification(
+        context: Context,
+        @DrawableRes smallIcon: Int,
+        data: Map<String?, String?>,
+    ): Boolean {
+        return showNotification(context, smallIcon, data, "Verloop Chat Message")
```

Comment:
Why is this hard coded?

Reply:
Why does the channel name consists of spaces?

Reply:
Because channel name will be visible to the user. It should be readable. What is hardcoded? Any suggestions here?